docs/quickstart.md

```diff
@@ -30,6 +30,12 @@ Learning any new authentication system can be difficult, especially as they get
     - [Creating Users](#creating-users)
     - [Deleting Users](#deleting-users)
     - [Editing a User](#editing-a-user)
+  - [Forcing Password Reset](#forcing-password-reset)
+    - [requiresPasswordReset()](#requirespasswordreset)
+    - [forcePaswordReset()](#forcepasswordreset)
+    - [undoForcePasswordReset()](#undoforcepasswordreset)
+    - [forceMultiplePasswordReset()](#forcemultiplepasswordreset)
+    - [forceGlobalPasswordReset()](#forceglobalpasswordreset)
```

Comment:
I think these sections are not good for the Quick Start page.
It seems better to move another page. A new page?

And the section titles are method names. I think it is better
that using what to do instead of the method name.